File: pgm/ABC_Java/ABC_322_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        reader.readLine();
        final String S = reader.readLine();
        final String ABC = "ABC";
        final int none = -1;
        System.out.println (S.indexOf (ABC) == none ? none
                                                    : S.indexOf (ABC) + 1);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_322_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.StringTokenizer

// import kotlin.math.*

const val ABC: String = "ABC"
const val one: Int = 1

fun main() {
    readln()
    val S: String = readln()
    if (S.indexOf(ABC) > -one) {
        println(S.indexOf(ABC) + one)
    } else {
        println(S.indexOf(ABC))
    }
    kotlin.system.exitProcess(0)
}